plugins/modules/helm_cli.py

```diff
@@ -0,0 +1,593 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli
+short_description: Manages Kubernetes packages with the Helm package manager
+description:
+  - Install, upgrade, delete packages with the Helm package manager.
+version_added: "2.10"
+author:
+  - Lucas Boisserie (@LucasBoisserie)
+  - Matthieu Diehr (@d-matt)
+requirements:
+  - "helm (https://github.com/helm/helm/releases)"
+  - "yaml (https://pypi.org/project/PyYAML/)"
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use.
+    required: false
+    type: path
+  chart_ref:
+    description:
+      - chart_reference on chart repository
+      - path to a packaged chart
+      - path to an unpacked chart directory
+      - absolute URL
+      - Required when I(release_state) is set to C(present)
+    required: false
+    type: path
+  chart_repo_url:
+    description:
+      - Chart repository URL where to locate the requested chart
+    required: false
+    type: str
+  chart_repo_username:
+    description:
+      - Chart repository username where to locate the requested chart
+      - Required if I(chart_repo_password) is specified
+    required: false
+    type: str
+  chart_repo_password:
```

Comment:
I would try to find another method to validate the chart repo username and password. Since this will expose sensitive information to the devices like `/proc/pid/cmdline`

Reply:
I closed this conversation to keep only one with fabianvf